source/geometry/Quaternion.ooc

```diff
@@ -28,15 +29,9 @@ Quaternion: cover {
 	y ::= this imaginary y
 	z ::= this imaginary z
 
-	// NOTE: The coordinates are represented differently in C# Kean:
-	// x = this w
-	// y = this x
-	// z = this y
-	// w = this z
-
 	isValid ::= this real isNumber && this imaginary isValid
 	isIdentity ::= this real equals(1.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
-	isNull ::= this real equals(0.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
+	isZero ::= this real equals(0.f) && this imaginary x equals(0.f) && this imaginary y equals(0.f) && this imaginary z equals(0.f)
```

Comment:
I think the most common name for a quaternion with all 0's is "Zero quaternion", but I may be wrong. I think it's pretty consistent with some of the other geometry types:
Zero vector - (0, 0, 0)
Zero matrix - (0, 0, 0, 0, ...)
Zero quaternion - (0, 0, 0, 0)